payload/bin/filesig.py

```
@@ -8,15 +8,23 @@
 
 
 def parseArgs(arguments=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)